src/cuda_zfp/cuZFP.cu

```diff
@@ -580,8 +580,9 @@ cuda_compress(zfp_stream *stream, const zfp_field *field, int variable_rate)
       cuZFP::chunk_process_launch((uint*)d_stream, d_offsets, i, cur_blocks, last_chunk, buffer_maxbits, num_sm);
     }
     // The total length in bits is now in the base of the prefix sum.
+    // but it excludes the 64-bit "add_padding". -> Pad to 64b, convert to bytes
     cudaMemcpy (&stream_bytes, d_offsets, sizeof (unsigned long long), cudaMemcpyDeviceToHost);
-    stream_bytes = (stream_bytes + 7) / 8;
+    stream_bytes = ((stream_bytes + 63) / 64) * 8;
```

Comment:
I don't think this was wrong in the first place.  This rounds the number of *bytes* (not bits) to the next multiple of 8 bytes = 64 bits.

Reply:
Sorry, I should have looked at the code above for context.  I guess `stream_bytes` initially is measured in number of bits, so this does do the correct thing.

Reply:
Correct, the comment states the offset is still in bits. I could have use 2 variables stream_bits and stream_bytes to make it more explicit.

Reply:
That being said, `stream_bytes` is not a great name for a bit offset.  A more appropriate name may promote code readability.